src/components/TextWrap/index.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import styled, { css } from 'styled-components';
+import { debounce } from 'ts-debounce';
+import Tooltip from '../Tooltip';
+
+type Props = {
+  children?: React.ReactNode;
+  isCapitalized?: boolean;
+  list?: string[];
+  listCount?: number;
+};
+
+const TextWrap = styled.span<Props>`
+  display: block;
+  margin-right: ${props => (props.listCount ? '40px;' : 0)};
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+
+  ${props =>
+    props.isCapitalized &&
+    css`
+      text-transform: capitalize;
+    `}
+
+  ${props =>
+    props.listCount &&
+    `
+    &:before {
+      background: ${props.theme.table.color.badge}
```

Comment:
use 'css' before the ` for better code highlighting and don't forget the ; at the end (so we don't get weird bugs like we had with the button)